service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -286,13 +288,13 @@ void shouldCreateExpectedMapWhenGivenPopulatedCaseData(GeneratedOrderType orderT
             caseData.getJudgeAndLegalAdvisor());
 
         assertThat(templateData).containsAllEntriesOf(expectedMap);
-        assertThat(templateData).containsKey("courtseal");
+        assertThat(templateData).containsEntry("courtseal", COURT_SEAL.getValue());
```

Comment:
In tests I would recommend not using the enums from the main code base, if we instead use the hard coded string values here then if someone accidentally changes the value in the enum the test should be able to pick it up

Reply:
Yup, I will change.